servo-git/.SRCINFO

```diff
@@ -1,14 +1,13 @@
 pkgbase = servo-git
 	pkgdesc = Parallel Browser Project: web browser written in Rust
-	pkgver = r47479.92196d985dc
+	pkgver = r48452.2ca6c4b52be
 	pkgrel = 1
 	url = https://github.com/servo/servo
 	install = servo-git.install
 	arch = x86_64
 	arch = i686
 	license = MPL
-	makedepends = autoconf2.13
-	makedepends = cargo
+	makedepends = rustup
```

Comment:
Why `rustup` as a build time dependency but exporting `RUSTUP_TOOLCHAIN=stable` in the build? If that variable works then we should be fine with the stable Rust/Cargo tooling right? If what we need is actually just other tools lets see if the other tools are packaged. It might be that `llvm-tolls-preview` does have something not otherwise packaged, in which case I'm fine with making this rustup based for now, but lets also figure out exactly what the MSRV is and what tooling we need, not just throw the book at it and let it download whatever it wants.

Thanks for taking the time to contribute though.

Reply:
I'm building with system rust now. Would it be ok to post build log as new issue "build servo with system rust" or just post in this PR?

Reply:
```
    Updating crates.io index
 Downloading crates ...
  Downloaded taplo-cli v0.9.0
  Installing taplo-cli v0.9.0
    Updating crates.io index
warning: package `ahash v0.7.6` in Cargo.lock is yanked in registry `crates-io`, consider running without --locked
warning: package `ahash v0.8.3` in Cargo.lock is yanked in registry `crates-io`, consider running without --locked
warning: package `crossbeam-channel v0.5.6` in Cargo.lock is yanked in registry `crates-io`, consider running without --locked
    Updating crates.io index
 Downloading crates ...
  Downloaded async-trait v0.1.58
  Downloaded addr2line v0.17.0
  Downloaded want v0.3.0
  Downloaded atty v0.2.14
  Downloaded autocfg v1.1.0
  Downloaded adler v1.0.2
  Downloaded webpki v0.22.2
  Downloaded foreign-types-shared v0.1.1
  Downloaded ansi_term v0.12.1
  Downloaded openssl-probe v0.1.5
  Downloaded pin-utils v0.1.0
  Downloaded num-traits v0.2.15
  Downloaded glob v0.3.0
  Downloaded futures-executor v0.3.25
  Downloaded proc-macro2 v1.0.66
  Downloaded memoffset v0.6.5
  Downloaded pin-project-lite v0.2.9
  Downloaded serde_spanned v0.6.5
  Downloaded num-rational v0.4.1
  Downloaded hyper-tls v0.5.0
  Downloaded beef v0.5.2
  Downloaded lsp-async-stub v0.6.1
  Downloaded logos-derive v0.12.1
  Downloaded mime v0.3.16
  Downloaded lru v0.7.8
  Downloaded http-body v0.4.5
  Downloaded tinyvec_macros v0.1.0
  Downloaded arc-swap v1.5.1
  Downloaded scopeguard v1.1.0
  Downloaded httparse v1.8.0
  Downloaded serde_urlencoded v0.7.1
  Downloaded pkg-config v0.3.25
  Downloaded proc-macro-error-attr v1.0.4
  Downloaded rustc-hash v1.1.0
  Downloaded native-tls v0.2.10
  Downloaded rustc-demangle v0.1.21
  Downloaded num-cmp v0.1.0
  Downloaded bitflags v1.3.2
  Downloaded miniz_oxide v0.5.4
  Downloaded untrusted v0.7.1
  Downloaded ipnet v2.5.0
  Downloaded unicode-width v0.1.10
  Downloaded version_check v0.9.4
  Downloaded fraction v0.13.1
  Downloaded serde_repr v0.1.9
  Downloaded spin v0.5.2
  Downloaded rustls-pemfile v1.0.1
  Downloaded schemars_derive v0.8.11
  Downloaded nu-ansi-term v0.46.0
  Downloaded url v2.3.1
  Downloaded tower-service v0.3.2
  Downloaded try-lock v0.2.3
  Downloaded http v0.2.8
  Downloaded hashbrown v0.12.3
  Downloaded time-macros v0.2.5
  Downloaded tap v1.0.1
  Downloaded semver v1.0.14
  Downloaded mio v0.8.5
  Downloaded smallvec v1.10.0
  Downloaded quote v1.0.21
  Downloaded proc-macro-error v1.0.4
  Downloaded once_cell v1.18.0
  Downloaded ryu v1.0.11
  Downloaded thiserror v1.0.37
  Downloaded futures-util v0.3.25
  Downloaded termcolor v1.1.3
  Downloaded strsim v0.10.0
  Downloaded sha1 v0.10.5
  Downloaded rowan v0.15.10
  Downloaded hashbrown v0.14.3
  Downloaded parking_lot_core v0.9.4
  Downloaded os_str_bytes v6.3.0
  Downloaded minimal-lexical v0.2.1
  Downloaded crossbeam-channel v0.5.6
  Downloaded jsonschema v0.17.1
  Downloaded fancy-regex v0.11.0
  Downloaded indexmap v2.1.0
  Downloaded slab v0.4.7
  Downloaded memchr v2.5.0
  Downloaded lsp-types v0.93.2
  Downloaded unicode-normalization v0.1.22
  Downloaded tracing-log v0.1.3
  Downloaded num v0.4.0
  Downloaded clap v3.2.23
  Downloaded unicode-ident v1.0.5
  Downloaded uuid v1.2.1
  Downloaded tracing-core v0.1.30
  Downloaded tracing v0.1.37
  Downloaded toml_edit v0.19.15
  Downloaded serde_json v1.0.87
  Downloaded regex-automata v0.1.10
  Downloaded rayon v1.5.3
  Downloaded winnow v0.5.35
  Downloaded tracing-subscriber v0.3.16
  Downloaded bstr v0.2.17
  Downloaded syn v1.0.103
  Downloaded regex-syntax v0.6.27
  Downloaded webpki-roots v0.22.5
  Downloaded rustls v0.20.7
  Downloaded regex v1.6.0
  Downloaded openssl v0.10.60
  Downloaded object v0.29.0
  Downloaded reqwest v0.11.12
  Downloaded tokio-util v0.7.4
  Downloaded time v0.3.16
  Downloaded idna v0.3.0
  Downloaded sharded-slab v0.1.4
  Downloaded libc v0.2.136
  Downloaded serde v1.0.147
  Downloaded tokio v1.24.2
  Downloaded gimli v0.26.2
  Downloaded num-bigint v0.4.3
  Downloaded vcpkg v0.2.15
  Downloaded typenum v1.15.0
  Downloaded toml v0.7.8
  Downloaded tinyvec v1.6.0
  Downloaded taplo v0.13.0
  Downloaded serde_derive v1.0.147
  Downloaded schemars v0.8.11
  Downloaded prettydiff v0.6.1
  Downloaded openssl-sys v0.9.96
  Downloaded unicode-bidi v0.3.8
  Downloaded tracing-attributes v0.1.23
  Downloaded tokio-rustls v0.23.4
  Downloaded thread_local v1.1.4
  Downloaded textwrap v0.16.0
  Downloaded socket2 v0.4.7
  Downloaded serde_derive_internals v0.26.0
  Downloaded sct v0.7.0
  Downloaded rayon-core v1.9.3
  Downloaded parking_lot v0.12.1
  Downloaded hyper v0.14.20
  Downloaded toml_datetime v0.6.5
  Downloaded tokio-native-tls v0.3.0
  Downloaded tokio-macros v1.8.0
  Downloaded thiserror-impl v1.0.37
  Downloaded text-size v1.1.0
  Downloaded taplo-common v0.5.0
  Downloaded overload v0.1.1
  Downloaded h2 v0.3.19
  Downloaded itertools v0.10.5
  Downloaded indexmap v1.9.1
  Downloaded crossbeam-epoch v0.9.11
  Downloaded json_value_merge v2.0.0
  Downloaded getrandom v0.2.8
  Downloaded encoding_rs v0.8.31
  Downloaded generic-array v0.14.6
  Downloaded futures-io v0.3.25
  Downloaded cc v1.0.73
  Downloaded hyper-rustls v0.23.0
  Downloaded base64 v0.21.2
  Downloaded num-iter v0.1.43
  Downloaded bitflags v2.4.1
  Downloaded bytes v1.2.1
  Downloaded base64 v0.13.1
  Downloaded percent-encoding v2.2.0
  Downloaded num_cpus v1.13.1
  Downloaded log v0.4.17
  Downloaded time-core v0.1.0
  Downloaded itoa v1.0.4
  Downloaded futures-sink v0.3.25
  Downloaded futures v0.3.25
  Downloaded hex v0.4.3
  Downloaded fnv v1.0.7
  Downloaded backtrace v0.3.66
  Downloaded iso8601 v0.6.1
  Downloaded either v1.8.0
  Downloaded httpdate v1.0.2
  Downloaded futures-channel v0.3.25
  Downloaded ahash v0.8.3
  Downloaded logos v0.12.1
  Downloaded lock_api v0.4.9
  Downloaded codespan-reporting v0.11.1
  Downloaded bytecount v0.6.3
  Downloaded block-buffer v0.10.3
  Downloaded anyhow v1.0.66
  Downloaded dyn-clone v1.0.9
  Downloaded clap_derive v3.2.18
  Downloaded matchers v0.1.0
  Downloaded lazy_static v1.4.0
  Downloaded globset v0.4.9
  Downloaded futures-macro v0.3.25
  Downloaded form_urlencoded v1.1.0
  Downloaded crypto-common v0.1.6
  Downloaded clap_lex v0.2.4
  Downloaded cfg-if v1.0.0
  Downloaded bit-set v0.5.3
  Downloaded openssl-macros v0.1.0
  Downloaded num_threads v0.1.6
  Downloaded heck v0.4.0
  Downloaded num-integer v0.1.45
  Downloaded futures-task v0.3.25
  Downloaded crossbeam-utils v0.8.12
  Downloaded ahash v0.7.6
  Downloaded digest v0.10.5
  Downloaded futures-core v0.3.25
  Downloaded crossbeam-deque v0.8.2
  Downloaded countme v3.0.1
  Downloaded nom v7.1.1
  Downloaded equivalent v1.0.1
  Downloaded bit-vec v0.6.3
  Downloaded foreign-types v0.3.2
  Downloaded cpufeatures v0.2.5
  Downloaded async-recursion v1.0.0
  Downloaded aho-corasick v0.7.19
  Downloaded ring v0.16.20
  Downloaded num-complex v0.4.2
   Compiling autocfg v1.1.0
   Compiling proc-macro2 v1.0.66
   Compiling quote v1.0.21
   Compiling unicode-ident v1.0.5
   Compiling syn v1.0.103
   Compiling cfg-if v1.0.0
   Compiling libc v0.2.136
   Compiling version_check v0.9.4
   Compiling once_cell v1.18.0
   Compiling memchr v2.5.0
   Compiling cc v1.0.73
   Compiling serde_derive v1.0.147
   Compiling serde v1.0.147
   Compiling scopeguard v1.1.0
   Compiling crossbeam-utils v0.8.12
   Compiling log v0.4.17
   Compiling rayon-core v1.9.3
   Compiling pin-project-lite v0.2.9
   Compiling either v1.8.0
   Compiling itoa v1.0.4
   Compiling smallvec v1.10.0
   Compiling parking_lot_core v0.9.4
   Compiling futures-core v0.3.25
   Compiling bytes v1.2.1
   Compiling tracing-core v0.1.30
   Compiling futures-sink v0.3.25
   Compiling futures-channel v0.3.25
   Compiling futures-task v0.3.25
   Compiling tinyvec_macros v0.1.0
   Compiling futures-util v0.3.25
   Compiling pkg-config v0.3.25
   Compiling ahash v0.7.6
   Compiling vcpkg v0.2.15
   Compiling tinyvec v1.6.0
   Compiling percent-encoding v2.2.0
   Compiling memoffset v0.6.5
   Compiling crossbeam-epoch v0.9.11
   Compiling rayon v1.5.3
   Compiling crossbeam-channel v0.5.6
   Compiling indexmap v1.9.1
   Compiling lock_api v0.4.9
   Compiling num_cpus v1.13.1
   Compiling getrandom v0.2.8
   Compiling crossbeam-deque v0.8.2
   Compiling hashbrown v0.12.3
   Compiling slab v0.4.7
   Compiling tokio v1.24.2
   Compiling socket2 v0.4.7
   Compiling mio v0.8.5
   Compiling parking_lot v0.12.1
   Compiling num-traits v0.2.15
   Compiling pin-utils v0.1.0
   Compiling futures-io v0.3.25
   Compiling ryu v1.0.11
   Compiling fnv v1.0.7
   Compiling serde_json v1.0.87
   Compiling ring v0.16.20
   Compiling unicode-normalization v0.1.22
   Compiling form_urlencoded v1.1.0
   Compiling num-integer v0.1.45
   Compiling unicode-bidi v0.3.8
   Compiling openssl-sys v0.9.96
   Compiling regex-syntax v0.6.27
   Compiling untrusted v0.7.1
   Compiling spin v0.5.2
   Compiling typenum v1.15.0
   Compiling http v0.2.8
   Compiling aho-corasick v0.7.19
   Compiling idna v0.3.0
   Compiling num-bigint v0.4.3
   Compiling generic-array v0.14.6
   Compiling openssl v0.10.60
   Compiling foreign-types-shared v0.1.1
   Compiling httparse v1.8.0
   Compiling foreign-types v0.3.2
   Compiling backtrace v0.3.66
   Compiling num-rational v0.4.1
   Compiling num-iter v0.1.43
   Compiling native-tls v0.2.10
   Compiling try-lock v0.2.3
   Compiling rustls v0.20.7
   Compiling bitflags v2.4.1
   Compiling adler v1.0.2
   Compiling gimli v0.26.2
   Compiling lazy_static v1.4.0
   Compiling miniz_oxide v0.5.4
   Compiling want v0.3.0
   Compiling http-body v0.4.5
   Compiling object v0.29.0
   Compiling proc-macro-error-attr v1.0.4
   Compiling regex v1.6.0
   Compiling ahash v0.8.3
   Compiling tower-service v0.3.2
   Compiling openssl-probe v0.1.5
   Compiling time-core v0.1.0
   Compiling anyhow v1.0.66
   Compiling rustc-demangle v0.1.21
   Compiling httpdate v1.0.2
   Compiling serde_derive_internals v0.26.0
   Compiling time-macros v0.2.5
   Compiling num-complex v0.4.2
   Compiling addr2line v0.17.0
   Compiling proc-macro-error v1.0.4
   Compiling beef v0.5.2
   Compiling schemars v0.8.11
   Compiling rustc-hash v1.1.0
   Compiling num_threads v0.1.6
   Compiling minimal-lexical v0.2.1
   Compiling countme v3.0.1
   Compiling async-trait v0.1.58
   Compiling bit-vec v0.6.3
   Compiling text-size v1.1.0
   Compiling encoding_rs v0.8.31
   Compiling thiserror v1.0.37
   Compiling rowan v0.15.10
   Compiling webpki v0.22.2
   Compiling sct v0.7.0
   Compiling bit-set v0.5.3
   Compiling nom v7.1.1
   Compiling num v0.4.0
   Compiling crypto-common v0.1.6
   Compiling block-buffer v0.10.3
   Compiling regex-automata v0.1.10
   Compiling bstr v0.2.17
   Compiling dyn-clone v1.0.9
   Compiling equivalent v1.0.1
   Compiling hashbrown v0.14.3
   Compiling bitflags v1.3.2
   Compiling overload v0.1.1
   Compiling base64 v0.13.1
   Compiling semver v1.0.14
   Compiling nu-ansi-term v0.46.0
   Compiling matchers v0.1.0
   Compiling rustls-pemfile v1.0.1
   Compiling globset v0.4.9
   Compiling indexmap v2.1.0
   Compiling digest v0.10.5
   Compiling fraction v0.13.1
   Compiling iso8601 v0.6.1
   Compiling webpki-roots v0.22.5
   Compiling fancy-regex v0.11.0
   Compiling sharded-slab v0.1.4
   Compiling tracing-log v0.1.3
   Compiling tracing-attributes v0.1.23
   Compiling tokio-macros v1.8.0
   Compiling futures-macro v0.3.25
   Compiling openssl-macros v0.1.0
   Compiling schemars_derive v0.8.11
   Compiling logos-derive v0.12.1
   Compiling thiserror-impl v1.0.37
   Compiling tracing v0.1.37
   Compiling serde_repr v0.1.9
   Compiling atty v0.2.14
   Compiling itertools v0.10.5
   Compiling thread_local v1.1.4
   Compiling arc-swap v1.5.1
   Compiling logos v0.12.1
   Compiling num-cmp v0.1.0
   Compiling os_str_bytes v6.3.0
   Compiling winnow v0.5.35
   Compiling bytecount v0.6.3
   Compiling cpufeatures v0.2.5
   Compiling heck v0.4.0
   Compiling ipnet v2.5.0
   Compiling futures-executor v0.3.25
   Compiling base64 v0.21.2
   Compiling futures v0.3.25
   Compiling termcolor v1.1.3
   Compiling mime v0.3.16
   Compiling tokio-util v0.7.4
   Compiling tokio-native-tls v0.3.0
   Compiling tokio-rustls v0.23.4
   Compiling uuid v1.2.1
   Compiling clap_derive v3.2.18
   Compiling sha1 v0.10.5
   Compiling clap_lex v0.2.4
   Compiling tracing-subscriber v0.3.16
   Compiling async-recursion v1.0.0
   Compiling lru v0.7.8
   Compiling ansi_term v0.12.1
   Compiling unicode-width v0.1.10
   Compiling hex v0.4.3
   Compiling glob v0.3.0
   Compiling textwrap v0.16.0
   Compiling strsim v0.10.0
   Compiling tap v1.0.1
   Compiling prettydiff v0.6.1
   Compiling codespan-reporting v0.11.1
   Compiling url v2.3.1
   Compiling time v0.3.16
   Compiling h2 v0.3.19
   Compiling serde_spanned v0.6.5
   Compiling serde_urlencoded v0.7.1
   Compiling toml_datetime v0.6.5
   Compiling toml_edit v0.19.15
   Compiling clap v3.2.23
   Compiling lsp-types v0.93.2
   Compiling jsonschema v0.17.1
   Compiling json_value_merge v2.0.0
   Compiling taplo v0.13.0
   Compiling toml v0.7.8
   Compiling hyper v0.14.20
   Compiling hyper-rustls v0.23.0
   Compiling hyper-tls v0.5.0
   Compiling reqwest v0.11.12
   Compiling taplo-common v0.5.0
   Compiling lsp-async-stub v0.6.1
   Compiling taplo-cli v0.9.0
    Finished `release` profile [optimized] target(s) in 2m 00s
  Installing /build/.cargo/bin/taplo
   Installed package `taplo-cli v0.9.0` (executable `taplo`)
warning: be sure to add `/build/.cargo/bin` to your PATH to be able to run the installed binaries
  Installing crown v0.1.0 (/build/servo-git/src/servo-git/support/crown)
   Compiling crown v0.1.0 (/build/servo-git/src/servo-git/support/crown)
error[E0463]: can't find crate for `rustc_ast`
  --> support/crown/src/main.rs:10:1
   |
10 | extern crate rustc_ast;
   | ^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_driver`
  --> support/crown/src/main.rs:11:1
   |
11 | extern crate rustc_driver;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_error_messages`
  --> support/crown/src/main.rs:12:1
   |
12 | extern crate rustc_error_messages;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_errors`
  --> support/crown/src/main.rs:13:1
   |
13 | extern crate rustc_errors;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_hir`
  --> support/crown/src/main.rs:14:1
   |
14 | extern crate rustc_hir;
   | ^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_infer`
  --> support/crown/src/main.rs:15:1
   |
15 | extern crate rustc_infer;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_interface`
  --> support/crown/src/main.rs:16:1
   |
16 | extern crate rustc_interface;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_lint`
  --> support/crown/src/main.rs:17:1
   |
17 | extern crate rustc_lint;
   | ^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_log`
  --> support/crown/src/main.rs:18:1
   |
18 | extern crate rustc_log;
   | ^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_middle`
  --> support/crown/src/main.rs:19:1
   |
19 | extern crate rustc_middle;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_session`
  --> support/crown/src/main.rs:20:1
   |
20 | extern crate rustc_session;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_span`
  --> support/crown/src/main.rs:21:1
   |
21 | extern crate rustc_span;
   | ^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_trait_selection`
  --> support/crown/src/main.rs:22:1
   |
22 | extern crate rustc_trait_selection;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0463]: can't find crate for `rustc_type_ir`
  --> support/crown/src/main.rs:23:1
   |
23 | extern crate rustc_type_ir;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
   |
   = help: maybe you need to install the missing components with: `rustup component add rust-src rustc-dev llvm-tools-preview`

error[E0425]: cannot find value `UNROOTED_MUST_ROOT` in this scope
  --> support/crown/src/unrooted_must_root.rs:24:34
   |
24 |     lint_store.register_lints(&[&UNROOTED_MUST_ROOT]);
   |                                  ^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `UNROOTED_MUST_ROOT` in this scope
   --> support/crown/src/unrooted_must_root.rs:191:25
    |
191 |                         UNROOTED_MUST_ROOT,
    |                         ^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `UNROOTED_MUST_ROOT` in this scope
   --> support/crown/src/unrooted_must_root.rs:216:33
    |
216 | ...                   UNROOTED_MUST_ROOT,
    |                       ^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `UNROOTED_MUST_ROOT` in this scope
   --> support/crown/src/unrooted_must_root.rs:253:29
    |
253 |                     cx.lint(UNROOTED_MUST_ROOT, "Type must be rooted", |lint| {
    |                             ^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `UNROOTED_MUST_ROOT` in this scope
   --> support/crown/src/unrooted_must_root.rs:262:25
    |
262 |                 cx.lint(UNROOTED_MUST_ROOT, "Type must be rooted", |lint| {
    |                         ^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `UNROOTED_MUST_ROOT` in this scope
   --> support/crown/src/unrooted_must_root.rs:293:21
    |
293 |                     UNROOTED_MUST_ROOT,
    |                     ^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `UNROOTED_MUST_ROOT` in this scope
   --> support/crown/src/unrooted_must_root.rs:337:25
    |
337 |                         UNROOTED_MUST_ROOT,
    |                         ^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `TRACE_IN_NO_TRACE` in this scope
  --> support/crown/src/trace_in_no_trace.rs:37:34
   |
37 |     lint_store.register_lints(&[&TRACE_IN_NO_TRACE, &EMPTY_TRACE_IN_NO_TRACE]);
   |                                  ^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `EMPTY_TRACE_IN_NO_TRACE` in this scope
  --> support/crown/src/trace_in_no_trace.rs:37:54
   |
31 | / const EMPTY_TRACE_IN_NO_TRACE_MSG: &str =
32 | |     "must_not_have_traceable marked wrapper is not needed for types that implements \
33 | | empty Traceable (like primitive types). Consider removing the wrapper.";
   | |________________________________________________________________________- similarly named constant `EMPTY_TRACE_IN_NO_TRACE_MSG` defined here
...
37 |       lint_store.register_lints(&[&TRACE_IN_NO_TRACE, &EMPTY_TRACE_IN_NO_TRACE]);
   |                                                        ^^^^^^^^^^^^^^^^^^^^^^^ help: a constant with a similar name exists: `EMPTY_TRACE_IN_NO_TRACE_MSG`

error[E0425]: cannot find value `EMPTY_TRACE_IN_NO_TRACE` in this scope
   --> support/crown/src/trace_in_no_trace.rs:135:29
    |
31  | / const EMPTY_TRACE_IN_NO_TRACE_MSG: &str =
32  | |     "must_not_have_traceable marked wrapper is not needed for types that implements \
33  | | empty Traceable (like primitive types). Consider removing the wrapper.";
    | |________________________________________________________________________- similarly named constant `EMPTY_TRACE_IN_NO_TRACE_MSG` defined here
...
135 |                               EMPTY_TRACE_IN_NO_TRACE,
    |                               ^^^^^^^^^^^^^^^^^^^^^^^ help: a constant with a similar name exists: `EMPTY_TRACE_IN_NO_TRACE_MSG`

error[E0425]: cannot find value `TRACE_IN_NO_TRACE` in this scope
   --> support/crown/src/trace_in_no_trace.rs:143:29
    |
143 | ...                   TRACE_IN_NO_TRACE,
    |                       ^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `CROWN_IS_NOT_USED` in this scope
  --> support/crown/src/crown_is_not_used.rs:15:34
   |
15 |     lint_store.register_lints(&[&CROWN_IS_NOT_USED]);
   |                                  ^^^^^^^^^^^^^^^^^ not found in this scope

error[E0581]: return type references lifetime `'tcx`, which is not constrained by the fn input types
  --> support/crown/src/common.rs:75:65
   |
75 | fn find_primitive_impls<'tcx>(tcx: TyCtxt<'tcx>, name: &str) -> impl Iterator<Item = DefId> + 'tcx {
   |                                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0581]: return type references an anonymous lifetime, which is not constrained by the fn input types
   --> support/crown/src/common.rs:193:54
    |
193 |     fn find_crates(tcx: TyCtxt<'_>, name: Symbol) -> impl Iterator<Item = DefId> + '_ {
    |                                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: lifetimes appearing in an associated or opaque type are not considered constrained
    = note: consider introducing a named lifetime parameter

Some errors have detailed explanations: E0425, E0463, E0581.
For more information about an error, try `rustc --explain E0425`.
error: could not compile `crown` (bin "crown") due to 28 previous errors
error: failed to compile `crown v0.1.0 (/build/servo-git/src/servo-git/support/crown)`, intermediate artifacts can be found at `/build/servo-git/src/servo-git/target`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
 * Setting up virtual environment...
 * Upgrading pip...
 * Installing Python requirements...
 * Installing taplo...
 * Installing crown (the Servo linter)...
Traceback (most recent call last):
  File "/build/servo-git/src/servo-git/./mach", line 48, in <module>
    main(sys.argv)
  File "/build/servo-git/src/servo-git/./mach", line 25, in main
    sys.exit(mach_bootstrap.bootstrap_command_only(topdir))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/build/servo-git/src/servo-git/python/mach_bootstrap.py", line 203, in bootstrap_command_only
    servo.platform.get().bootstrap(force, skip_platform)
  File "/build/servo-git/src/servo-git/python/servo/platform/base.py", line 62, in bootstrap
    installed_something |= self.install_crown(force)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/build/servo-git/src/servo-git/python/servo/platform/base.py", line 80, in install_crown
    raise EnvironmentError("Installation of crown failed.")
OSError: Installation of crown failed.
==> ERROR: A failure occurred in build().
    Aborting...
```

Reply:
In there you can see this `rustup component add rust-src rustc-dev llvm-tools-preview` that I put in PKGBUILD.